docs/docs/getting-started/quickstart.md

```
@@ -1,204 +1,114 @@
 npm install -g altimate-code
 ```
 
-> **Zero additional setup.** One command install.
+The install runs a postinstall script that links the platform-specific binary onto your PATH, sets up the bundled dbt tooling, and copies the built-in skills into `~/.altimate/builtin`.
 
----
+??? note "Does your org restrict npm install scripts?"
+    If your org blocks npm lifecycle scripts (for example with `--ignore-scripts` or a script-approval gate like `@lavamoat/allow-scripts`), that postinstall step won't run, and the CLI may not function correctly. To let it run those postinstall scripts, run this and reinstall:
 
-## Step 2: Configure Your LLM
+    ```bash title="Terminal"
```

markdownlint-cli2 (0.23.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)